### ISSUE TITLE
Dialog jump to bottom on open

### ISSUE THREAD

**Issue by neobie** [MohammadYounes/AlertifyJS#212]
May I know why the dialog jump to bottom by default ? (With multilines long text)
How do I prevent the scrolling to bottom on open ?
Using 1.11.4

https://codepen.io/anon/pen/BgLGVq

**Comment by MohammadYounes** [comment]
The jump you see is due to focusing the OK button. The built in confirm dialog implements a [`defaultFocus`](https://alertifyjs.com/confirm/defaultFocus). 


Currently, it has to be either OK or Cancel ([it can't be disabled](https://github.com/MohammadYounes/AlertifyJS/blob/master/src/js/confirm.js#L151)). I believe allowing end user to disable it completely is a good enhancement.

For the time being, you can create a custom confirm inherited from the builtin one, but without default focus, see https://codepen.io/anon/pen/ZdpdmG?editors=1010

**Comment by anotherCoward** [comment]
It's not perfect, but don't wanna change everything rule just for this example.

But with some flex-CSS Rules, the size could match the screen always and allow scrolling at the body of the dialogs, without modifying the script itself and keep focus on okay.

```css
.ajs-dialog {
  max-height: 60vh; /* max allowed height would be 60% of the current viewport height */
  display: flex;
  flex-direction: column;
  flex: 1 1 auto;
}
.ajs-header,
.ajs-footer {
  position: relative;
}
.ajs-body {
  position: relative;
  display: flex;
  flex: 1 1 auto;
  height: 100%;
  width: 100%;
  overflow: auto;
}
.ajs-content {
  position: relative;
  display: flex;
  flex: 1 0 auto;
  width: 100%;
  height: 100%;
}```

**Comment by MohammadYounes** [comment]
Closed in 05c9048a7a1e0e1d2545cd72719ba307a724cf06 

See https://alertifyjs.com/confirm/defaultFocusOff
